timm/layers/typing.py

```diff
@@ -0,0 +1,9 @@
+import functools
+import types
+from typing import Tuple, Union
+
+import torch.nn
+
+
+LayerType = Union[type, str, types.FunctionType, functools.partial, torch.nn.Module]
```

Comment:
as with the Type[nn.Module] in the ResNet, this also needs to change right?

This would suffice right?
`LayerType = Union[str, Callable, Type[torch.nn.Module]]`

Reply:
Ah, my bad. I forgot the change to Type here.

Reply:
I think we don't need the extra type, types.FunctionTYpe, partial, etc either, str + Callable + Type[nn.Module] should cover it all